README.md

```diff
@@ -46,16 +46,16 @@ export default Ember.Component.extend({
 You replace it with this:
 
 ```javascript
-import Ember from 'ember'
+import Component from '@ember/component'
 import { action, computed } from 'ember-decorators/object';
 import { service } from 'ember-decorators/service';
 
-export default class MyComponent extends Ember.Component {
```

Comment:
I think we should encourage users to name their classes, it's one of the benefits of ES Classes (easier to grep for class names, no need for a class to be resolved to know its type - this plagues us with classes we use that are not resolved)